include/boost/geometry/algorithms/discrete_hausdorff_distance.hpp

```diff
@@ -264,7 +264,9 @@ struct discrete_hausdorff_distance<multi_linestring1, multi_linestring2, multi_l
 // Algorithm overload using explicitly passed Pt-Pt distance strategy
 
 /*!
-\brief calculate discrete hausdorff distance between two geometries ( currently works for LineString-LineString, MultiPoint-MultiPoint, Point-MultiPoint, MultiLineString-MultiLineString ) using specified strategy
+\brief Calculate discrete hausdorff distance between two geometries (currently
```

Comment:
I think it should be capitalised  `Hausdorff`

Reply:
Right, thanks.

Reply:
I also fixed Frechet distance the same way.